### ISSUE TITLE
🛠️ FIX : Fuzz Testing in PRBMath.t.sol

### ISSUE THREAD

**Issue by 0xpanicError** [BlocSoc-iitr/HorseRiders#24]
### 🛠️ [FIX]
Description:
PRBMath functions are not fuzz tested.

Objective:
Write fuzz tests for functions implemented in `src/ComplexHuff/PRBMath.huff`

**Comment by CanonicalJP** [comment]
@0xpanicError can I work on this issue?

**Comment by 0xpanicError** [comment]
Yes ofc. There is another PR working on fuzz testing so ensure the functions you work on don't overlap with that. 
https://github.com/BlocSoc-iitr/HorseRiders/pull/30
